src/caselawclient/Client.py

```diff
@@ -753,6 +771,19 @@ def get_properties_for_search_results(self, judgment_uris: list[str]) -> str:
         response = self._send_to_eval(vars, "get_properties_for_search_results.xqy")
         return decode_multipart(response)
 
+    def search_and_parse_results(
```

Comment:
@jacksonj04 @dragon-dxw 

At first, I wasn't sure about placing the 2 new methods `search_and_parse_results` `search_judgments_and_parse_results` in the Client library.

But after seeing that we have a mixture of methods, some that return:
- raw response objects
- text from decoded multipart responses
- other values given parsing of the decoded text

as well as the more boilerplatey eval, invoke etc methods, I though that although I think the class is probably doing too much at the moment, these 2 new methods do fit in with bulletpoint 3.

However, I do think at some point we should think about breaking this class out - I've had some thoughts on various options but think it needs some proper discussion.

Reply:
I've got some work in-flight at the moment which tackles this problem from the other end, ie "we should be able to take a bunch of stuff from a response and always get a coherent object or set of objects", so this will (I think) neatly meet up in the middle.